centreon/packaging/centreon-poller.yaml

```diff
@@ -120,15 +118,14 @@ overrides:
       - centreon-poller-centreon-engine
     conflicts:
       - centreon-poller-nagios
+      - centreon-broker-cbmod
```

Comment:
since centreon-engine-daemon already does the conflict + replace, I am not sure that this is needed here